docs/tutorials/local_sine_tutorial.rst

```diff
@@ -233,9 +233,9 @@ That's it! Now that these files are complete, we can run our simulation.
 
   $ python3 calling_script.py
 
-If everything ran perfectly and you included the above print-statements, You
+If everything ran perfectly and you included the above print statements, you
 should get something similar to the following output for ``H``. The columns might
-be rearranged.
+be rearranged as follows.
```

Comment:
(the columns may be arranged differently) - ie. this is saying that the order of fields shown below is not necessarily going to be the same.

Reply:
Doesn't the "might be rearranged as follows" imply this?

Reply:
The order of the columns don't appear to be predictable. The provided H sample is just one of many possible combinations. The order "might" be arranged "as follows", but it's a small chance.

Maybe "The output columns may be arranged differently." ? I'm trying to communicate the impression that as long as all rows and columns are included everything worked fine, and columns being rearranged into whichever-order is normal.